ui/packages/catalog-ui-search/src/main/webapp/react-component/container/sharing/sharing.tsx

```diff
@@ -128,39 +128,58 @@ export class Sharing extends React.Component<Props, State> {
     const loadingView = new LoadingView()
     this.attemptSave(attributes, usersToUnsubscribe)
       .then(() => {
```

Comment:
✏️ While I'm not opposed to this here, I think the `await` with `try/catch` would be easier to read.
```
try {
   await this.attemptSave(...)
   announcement.announce(...)
} catch (error) {
   if(error.message === ...) {
      ...
   }
}
```